main.cpp

```diff
@@ -73,12 +23,23 @@ void testPairToNumber(
     assert(pairNumber == expectedPairNumber);
 }
 
+void ToString()
+{
```

Comment:
ToString is actually doing 'to console' here. It needs to make one big string with all the reference entries for the manual.